src/audio/kpb.c

```diff
@@ -75,7 +75,11 @@ static size_t kpb_allocate_history_buffer(struct comp_data *kpb);
 static void kpb_clear_history_buffer(struct hb *buff);
 static void kpb_free_history_buffer(struct hb *buff);
 static bool kpb_has_enough_history_data(struct comp_data *kpb,
-					    struct hb *buff, size_t his_req);
+					struct hb *buff, size_t his_req);
+static void kpb_copy_sw16_ch2(struct comp_buffer *sink,
```

Comment:
Could we change function names to be aligned with the rest of processing components?
Also I don't think we need to specify number of channels, since your functions are universal.
`kpb_copy_s16`
`kpb_drain_s16`

Reply:
I have yet another idea, lets call them `kpb_copy_samples` and `kpb_drain_samples` and inside these function decide what buffer_read/write API to choose by format. Although for this PR selecting different samples format is out of scope as it is will be covered in different PR.